docker-compose.yml

```diff
@@ -33,6 +33,15 @@ services:
       - ${PWD}/.docker/vhost.conf:/etc/nginx/conf.d/default.conf:ro
       - ./:/app:delegated
 
+  nginx2:
```

Comment:
For consistency, consider calling the other `nginx` container `nginx1` (or `nginx0` if we count from 0).

Reply:
Fixed :-)